tests/plugins/test_koji_upload.py

```diff
@@ -711,11 +711,27 @@ def validate_buildroot(self, buildroot):
         assert set(osbs.keys()) == set([
             'build_id',
             'builder_image_id',
+            'koji',
         ])
 
         assert is_string_type(osbs['build_id'])
         assert is_string_type(osbs['builder_image_id'])
 
+        koji = osbs['koji']
+        assert isinstance(koji, dict)
+        assert set(koji.keys()) == set([
```

Comment:
JFYI, you can create set directly `{'item1', 'item2'}` without usage of an extra list. (Except empty set, `empty_set = set()`, because `{}` is empty dict)

Reply:
Yes, but I'm following the idiom of the rest of the function.

Reply:
I think it was originally done this way for Python 2.6 compatibility, which we no longer need.

IMHO we should follow the existing style in the commit making the change, and have a separate PR (not needed for this story) for using the "set literal" syntax throughout atomic-reactor where appropriate.